nodejs/week1/avg.js

```diff
@@ -0,0 +1,14 @@
+const args = process.argv.slice(2);
+const numbers = args.map(Number).filter(num => !isNaN(num));
```

Comment:
Very nice to use explaining variables to make your code clearer. I like the fact that you handled the case that the supplied arguments are not numbers, and you ignore those arguments. 👍